docs/README.md

```diff
@@ -1,13 +1,25 @@
 # Documentation For Developers
 
-Here are some documents important for TEAMMATES developers.
+Welcome to the documentation of the TEAMMATES project.
+
+Although it may look intimidating, the documentation will make working on the project much easier. Read through them well and it'll make contributing to TEAMMATES that much more enjoyable!
+
+## Contributing
+
+ If you are new and want to contribute but don't know where to start, this is the right place:
+
+1. [**Get Started**](contributors-guide.md): Step-by-step guide on how to get started.
+2. [**Setting Up**](setting-up.md): How to set up a local development environment
```

Comment:
Actually, what about the orientation guide? Remember it already contains some of these steps here, so maybe some of these can be replaced with a link to it instead?